`make build` causes `ERROR 403: rate limit exceeded.`

Seems that the error `ERROR 403: rate limit exceeded.` may occur when building kernel.

Do we have any good solutions?

```shell
root@azpvdrefgpu02:~/asterinas# make build
Cloning into '.'...
remote: Enumerating objects: 9, done.
remote: Counting objects: 100% (9/9), done.
remote: Compressing objects: 100% (8/8), done.
remote: Total 9 (delta 0), reused 0 (delta 0), pack-reused 0
Receiving objects: 100% (9/9), 14.94 KiB | 81.00 KiB/s, done.
CC <= /root/asterinas/regression/apps/execve/../../build/initramfs/regression/execve/execve
CC <= /root/asterinas/regression/apps/execve/../../build/initramfs/regression/execve/hello
CC <= /root/asterinas/regression/apps/fork/../../build/initramfs/regression/fork/fork
CC <= /root/asterinas/regression/apps/fork_c/../../build/initramfs/regression/fork_c/fork
CC <= /root/asterinas/regression/apps/getpid/../../build/initramfs/regression/getpid/getpid
CC <= /root/asterinas/regression/apps/hello_c/../../build/initramfs/regression/hello_c/hello
CC <= /root/asterinas/regression/apps/hello_pie/../../build/initramfs/regression/hello_pie/hello
CC <= /root/asterinas/regression/apps/hello_world/../../build/initramfs/regression/hello_world/hello_world
wget --header="Accept: application/vnd.github.v3.raw" \
        -O /root/asterinas/regression/apps/mongoose/../../build/mongoose/mongoose.c "https://api.github.com/repos/cesanta/mongoose/contents/mongoose.c?ref=7.13"
--2024-04-30 07:47:53--  https://api.github.com/repos/cesanta/mongoose/contents/mongoose.c?ref=7.13
Resolving xx.com (xx.com)... xx.xxx.xxx.xx
Connecting to xx.com (xx.com)|xx.xxx.xxx.xx|:xx... connected.
Proxy request sent, awaiting response... 403 rate limit exceeded
2024-04-30 07:47:54 ERROR 403: rate limit exceeded.

make[3]: *** [Makefile:31: /root/asterinas/regression/apps/mongoose/../../build/mongoose/mongoose.c] Error 8
make[2]: *** [Makefile:33: mongoose] Error 2
make[1]: *** [Makefile:76: /root/asterinas/regression/build/initramfs/regression] Error 2
make: *** [Makefile:121: initramfs] Error 2
root@azpvdrefgpu02:~/asterinas#
```

--- a/regression/apps/Makefile
+++ b/regression/apps/Makefile
@@ -10,7 +10,8 @@ REGRESSION_BUILD_DIR ?= $(INITRAMFS)/regression
 
 TEST_APPS := signal_c pthread network hello_world hello_pie hello_c fork_c fork execve pty mongoose
 
-C_SOURCES := $(shell find . -type f \( -name "*.c" -or -name "*.h" \) )
+# The C head and source files of all the apps, excluding the downloaded mongoose files
+C_SOURCES := $(shell find . -type f \( -name "*.c" -or -name "*.h" \) ! -name "mongoose.c" ! -name "mongoose.h")
 
 .PHONY: all
 all: $(TEST_APPS) scripts

--- a/regression/apps/mongoose/.gitignore
+++ b/regression/apps/mongoose/.gitignore
@@ -1,0 +1,1 @@
+mongoose.*

--- a/regression/apps/mongoose/Makefile
+++ b/regression/apps/mongoose/Makefile
@@ -2,7 +2,7 @@
 
 CUR_DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 BUILD_DIR := $(CUR_DIR)/../../build/initramfs/regression/network
-MONGOOSE_DIR := $(CUR_DIR)/../../build/mongoose
+MONGOOSE_DIR := $(CUR_DIR)
 MONGOOSE_C := $(MONGOOSE_DIR)/mongoose.c
 MONGOOSE_H := $(MONGOOSE_DIR)/mongoose.h
 MONGOOSE_FILES := $(MONGOOSE_C) $(MONGOOSE_H)
@@ -28,8 +28,7 @@ $(MONGOOSE_O): $(MONGOOSE_FILES)
 	$(CC) -c $(MONGOOSE_C) $(CFLAGS) -o $@
 
 $(MONGOOSE_FILES): | $(MONGOOSE_DIR)
-	wget --header="Accept: application/vnd.github.v3.raw" \
-		-O $@ "https://api.github.com/repos/cesanta/mongoose/contents/$(notdir $@)?ref=7.13"
+	wget -O $@ "https://raw.githubusercontent.com/cesanta/mongoose/7.13/$(notdir $@)"
 
 $(BUILD_DIR) $(MONGOOSE_DIR):
 	@mkdir -p $@

